package.json

```diff
@@ -26,7 +26,7 @@
   "devDependencies": {
     "lodash.times": "^3.0.1",
     "combokeys": "^2.4.3",
-    "edited": "^1.1.1",
+    "edited": "^1.1.2",
```

Comment:
Minor quibble, but I would say that this is more of a new feature in `edited` than a bugfix, so I probably would have bumped it to `1.2.0`.